### ISSUE TITLE
idea: HTML export with images

### ISSUE THREAD

**Issue by edmundlaugasson** [aardappel/treesheets#84]
The idea is related with "_HTML (Tables+Styling)_" and this is very nice export feature.
Would propose to probably create a separate choice "_HTML (Tables+Styling+Images)_".

Basically this means that <img> tag would be included in .html and image with relative path shown in HTML export. That's all.

**Comment by gwvo** [comment]
Agree that would be cool to have.

**Comment by iancustoica** [comment]
I would also love to see some form of html export that would include images... The only way I have found is to actually put the images in the same folder or a different folder as the html file and then manually edit the html file and add the img tag. However if I don't have the image at hand I got no idea how to obtain images in the html export.

**Comment by calvadosxo** [comment]
Yes I also think it's actually a deal breaker for me, literally the only thing keeping me from using this otherwise amazing app, since I need to edit the files in Android and HTML is perfect for that.

**Comment by StaffanBetner** [comment]
This is a feature I would like very much. It is a bit much to export an enormous image for my projects.

**Comment by aardappel** [comment]
This should not be particularly hard to add..

**Comment by StaffanBetner** [comment]
Famous last words :) I guess the main choice is to whether embed the images in the HTML file or have them separately?

**Comment by tobiolo** [comment]
Done
